### PR TITLE
Feature/yamlsupport

### PR DESCRIPTION
**Pull request by nyuuyn**
#### Short Description
Enable general usage of yaml tosca applications

#### Proposed Changes
- [x] Enable importing YAML csars via given winery importers
- [ ] Enable PlanBuilder to work with YAML csars
- [ ] Probably refactor code that is hard-wired to the XML spec of TOSCA
- [ ] Write Test Case for a YAML CSAR from https://github.com/OpenTOSCA/tosca-definitions-yaml-test-applications

### CODE REVIEW

**Comment by nyuuyn** [comment]
[DeploymentTestAgent.csar.zip](https://github.com/OpenTOSCA/container/files/8333588/DeploymentTestAgent.csar.zip)

Here is a small example, maybe not deployable, test csar for this PR.
Don't forget to rename it from *.csar.zip to *.csar as github doesn't allow .csar endings to be uploaded

**Comment by nyuuyn** [comment]
They whole PR is heavily depending on the following PR: https://github.com/eclipse/winery/pull/665

File: org.opentosca.container.core/src/main/resources/application.properties [hunk 1]
```diff
@@ -1,31 +1,31 @@
 # Container Configuration
 # Your external IP address, e.g. 192.168.0.131
-org.opentosca.container.hostname=localhost
+org.opentosca.container.hostname=192.168.0.239
```
Comment (lharzenetter) [inline org.opentosca.container.core/src/main/resources/application.properties:3]: revert these changes

**Review by lharzenetter — CHANGES_REQUESTED** [review]

**Comment by lharzenetter** [comment]
Is the PR in eclipse ready to be merged?

**Pull request closed without merging**